net/sched/act_skbmod.c

```diff
@@ -72,6 +73,25 @@ static int tcf_skbmod_run(struct sk_buff *skb, const struct tc_action *a,
 		ether_addr_copy(eth_hdr(skb)->h_source, (u8 *)tmpaddr);
 	}
 
+	wlen = skb_network_offset(skb);
+	if (flags & SKBMOD_F_INHERITDSFIELD) {
```

Comment:
Move the declaration **and** initialization of `wlen` inside of the `if`; sorry, it's my fault.